pkg/cgroup/cgroup.go

```diff
@@ -0,0 +1,292 @@
+package cgroup
+
+import (
+	"bufio"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/aquasecurity/tracee/pkg/mount"
+)
+
+type Cgroup interface {
+	initialize() error
```

Comment:
Since interfaces deal with what is publicly available, it's a bit unnecessary to have `initalize` here, and i'll expand further about something that may be simpler to do.